dts/bindings/timer/litex,machine-timer.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  LiteX Machine Timer
+
+  The LiteX machine timer provides RISC-V prigileved mtime and mtimecmp
+  registers.
```

Comment:
`litex_vexriscv` doesn't fully follow the RISC-V privileged spec and doesn't use the RISC-V machine timer driver.

Reply:
Hmm, I don't remember why I had added that line of code 3 years ago when implementing the initial irqc driver, but `litex_vexriscv` doesn't use this IRQ at all (tested on HW). So I'd suggest just removing the reference to `RISCV_MACHINE_TIMER_IRQ` and just leaving the `MACHINE_EXT` one.

Reply:
removed